src/scenes/GameMaster.cpp

```diff
@@ -0,0 +1,203 @@
+#include <smk/Color.hpp>
+#include <smk/Shape.hpp>
+#include <smk/Window.hpp>
+#include <smk/Input.hpp>
+#include <smk/Rectangle.hpp>
+#include <smk/Vertex.hpp>
+#include <smk/VertexArray.hpp>
+#include <smk/Text.hpp>
+
+#include "headers/GameMaster.hpp"
+#include "headers/Button.hpp"
+#include "headers/SceneManager.hpp"
+
+#include <iostream>
+#include <map>
+#include <string>
+#include <locale>
+#include <codecvt>
+#include <algorithm>
+
+GameMaster::GameMaster(smk::Window &window) : window(window) {
+  this->inputBox = new InputBox(500, 150, 800, 50, window, 48);
+  inputBox->setColor(smk::Color::RGB(0.6, 0.4, 0.6));
+  page = 0;
+  words = 
```

Comment:
I don't get it. Do we have to manually add those words? What about working with presets?

Reply:
It's there only for testing purposes. At the time I was making it, you haven't commited map with your presets yet so I made my own. When we merge it, I will remove it.